codesigndoc/codesigngroup.go

```diff
@@ -81,7 +81,9 @@ Would you like to use this team to sign your project?`, archiveCertificate.TeamI
 	if !useArchiveTeam {
 		teams := []string{}
 		for team := range certificatesByTeam {
-			teams = append(teams, team)
+			if teamHasCertificates(archiveCertificate, certificatesByTeam[team]) {
```

Comment:
the initial logic here is not correct, this function should work like:

1. Do you wasnt to use the archive team for archive? (if multiple team's identity could create this archive)
  `no` -> list the teams that could create this archive and let select
2. Ipa export method?
3. Do you want to use the archive team for the ipa export (if multiple team's identity could export the archive)
  `no` -> list the teams that could export this archive and let select

please update the whole function